Firestore/core/src/firebase/firestore/core/query.h

```diff
@@ -39,6 +39,8 @@ namespace core {
 
 using CollectionGroupId = std::shared_ptr<const std::string>;
 
+enum class LimitType { First, Last, None };
```

Comment:
If you put the `None` value first in the list it will be the one that gets the zero value. Since this is a primitive value that's not as useful as it could be (because primitives don't normally get default initialized) but in cases where memory is zeroed out it's useful for that to create a `None`.

Reply:
> LGTM, with a minor suggestion and a last straggling maybe issue.

The issues raised are now fixed.